### PR TITLE
    doc: update documentation and grub file

### PR DESCRIPTION
**Pull request by jackwhich**
1. Because the < > in the document will mislead users to copy and paste directly,
    modify the < > to " ".
2. Add sample diagrams for user reference.
3. Because there is a problem with the user adding the GRUB_CMDLINE_LINUX=text
    parameter, delete this parameter.
4. The script generation of launch_uos_id3.sh does not match the GitHub and doc,
    modify the parameters.

    Signed-off-by: zhongzhenx.liu <zhongzhenx.liu@intel.com>

### CODE REVIEW

**Review by gvancuts — CHANGES_REQUESTED** [review]
Please fix your commit message, especially the first paragraph: really hard to understand and various typos are in there.

File: doc/getting-started/getting-started.rst [hunk 1]
```diff
@@ -637,9 +637,9 @@ In the following steps, you will configure GRUB on the target system.
            insmod gzio
            insmod part_gpt
            insmod ext2
-           search --no-floppy --fs-uuid --set <UUID>
+           search --no-floppy --fs-uuid --set "3cac5675-e329-4cal-b346-0a3e65f99016"
```
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:640]: This isn't going to work.  The instructions above this code block say to

```
Replace ``<UUID>`` and ``<PARTUUID>`` with the output from the previous step.
```
but now the code block doesn't have those strings in it any more.  What we need is to put those UUID and PARUUID strings back in this code block, and then create another code block showing what a properly edited GRUB file example would look like.

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/getting-started/getting-started.rst (hunk 1 above)
Comment (jackwhich) [reply doc/getting-started/getting-started.rst]: already edited

**Review by dbkinder — APPROVED** [review]
LGTM, thanks

**Review by gvancuts — APPROVED** [review]
LGTM

**Pull request merged**